cloud-media-livestream/keypublisher/clients/fake_client.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""File containing a fake CPIX client for demonstration purposes."""
+
+import secrets
+
+from typing import List
+
+from . import cpix_client
+
+
+class FakeClient(cpix_client.CpixClient):
+    """Fake CPIX client, for demonstration purposes only."""
+
+    def fetch_keys(self, media_id: str, key_ids: List[str]):
+        """Generates random key information.
+
+        Args:
+        media_id (string): Name for your asset, sometimes used by DRM providers to
+            show usage and reports.
+        key_ids (string[]): List of IDs of any keys to fetch and prepare.
+
+        Returns:
+        dict: Object containing key information to be written to Secret Manager.
```

Comment:
```suggestion
            Object containing key information to be written to Secret Manager.
```

Reply:
I've modified the description here to clarify that the return is dictionary type with a mapping of key ID to json formatted object containing key info.

Reply:
The docstring should be consistent - either always include the type info in returns with the following format:

```
Returns:
    dict: ...
```
or
```
Returns:
    ...
```